src/libcollections/vec.rs

```diff
@@ -1627,6 +1630,7 @@ impl<T> IntoIter<T> {
     #[inline]
     /// Drops all items that have not yet been moved and returns the empty vector.
     #[unstable(feature = "iter_to_vec")]
+    #[deprecated(since = "1.3.0", reason = "replaced by drain()")]
```

Comment:
I don't see how this is replaced by drain, it may in fact be very useful (I'd use it in itertools if it were stable).